### PR TITLE
Final cli design

### LINKED ISSUE #50: final CLI design
I propose to use commands:
```
$ colin check
$ colin list-checks
$ colin list-configurations
```

How do we name configurations? Targets maybe?

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
Resolves #50

### CODE REVIEW

**Comment by lachmanfrantisek** [comment]
@TomasTomecek  I've found the solution to make the `check` default command, but the code is not nice. WDYT?

File: colin/cli/colin.py [hunk 1]
```diff
@@ -151,5 +182,13 @@ def _print_checks(checks):
             click.echo(str(check))
 
 
+def _get_log_level(debug, verbose):
+    if debug:
+        return logging.DEBUG
+    elif verbose:
+        return logging.INFO
```
Comment (jpopelka) [inline colin/cli/colin.py]: Just wondering whether it makes sense to specify `--debug` and `--verbose` at the same time, I mean what about:

```python
if debug and verbose:
    raise click.BadOptionUsage("Options '--debug' and '--verbose' cannot be used together.")
```

**Comment by TomasTomecek** [comment]
Something's wicked going on:
```
$ python3 -m colin.cli.colin check -f ./config/redhat.json 17d517eb05a1
16:27:24.398 target.py         WARNING Target is neither image nor container.
An error occurred: ColinException('Target type not found.',)
Error: Target type not found.
```

So I tried harder:
```
$ python3 -m colin.cli.colin check --debug -f ./config/redhat.json our.registry:8888/long-long/image-name:long-long-long-123-candidate-20151210143043
16:28:24.971 colin.py          DEBUG  Checking started.
16:28:24.971 target.py         DEBUG  Finding target '...'.
16:28:24.971 backend.py        INFO   conu has initiated, welcome to the party!
16:28:24.971 backend.py        DEBUG  conu version: 0.2.0
16:28:24.992 __init__.py       INFO   docker environment info: 'Client:\n Version:         1.13.1\n API version:     1.26\n Package version: docker-1.13.1-51.git4032bd5.fc29
.x86_64\n Go version:      go1.10\n Git commit:      3732dc7-unsupported\n Built:           Wed Mar 28 13:51:51 2018\n OS/Arch:         linux/amd64\n\nServer:\n Version:
     1.13.1\n API version:     1.26 (minimum version 1.12)\n Package version: docker-1.13.1-51.git4032bd5.fc29.x86_64\n Go version:      go1.10\n Git commit:      3732dc7-un
supported\n Built:           Wed Mar 28 13:51:51 2018\n OS/Arch:         linux/amd64\n Experimental:    false\n'
16:28:25.010 image.py          DEBUG  pull policy set to 'never'
16:28:25.014 target.py         WARNING Target is neither image nor container.
16:28:25.015 target.py         DEBUG  Target type not found.
```

**Comment by TomasTomecek** [comment]
CLI design LGTM

File: colin/cli/colin.py (hunk 1 above)
Comment (TomasTomecek) [reply colin/cli/colin.py]: +1

**Comment by lachmanfrantisek** [comment]
> Something's wicked going on:

It's a bad prove, but it works on my side..;-) Do you have a fresh version? I'll test it in some virtual environment.

File: colin/cli/colin.py (hunk 1 above)
Comment (lachmanfrantisek) [reply colin/cli/colin.py]: +1

**Comment by lachmanfrantisek** [comment]
@TomasTomecek  You definitely have an old version -- the debug messages do not correspond to the actual state.

**Comment by TomasTomecek** [comment]
Action item for me: write a serious script to check out pull requests locally and ensure they are updated.

**Comment by lachmanfrantisek** [comment]
@TomasTomecek  @jpopelka 

Thank you for the review.

-----

@TomasTomecek  I'm curious about the script...;-)

**Pull request merged**